eo/focus-ios.xliff

```diff
@@ -376,11 +376,39 @@
         <target>Fermi</target>
         <note>Button label used to close a menu that displays as a popup.</note>
       </trans-unit>
+      <trans-unit id="NewOnboarding.Button.Title.V2" xml:space="preserve">
+        <source>Get Started</source>
+        <note>Text for a label that indicates the title of button from onboarding screen version 2.</note>
+      </trans-unit>
+      <trans-unit id="NewOnboarding.Subtitle.V2" xml:space="preserve">
+        <source>Fast. Private. No Distractions.</source>
+        <note>Text for a label that indicates the subtitle for the onboarding screen version 2.</note>
+      </trans-unit>
       <trans-unit id="Onboarding.Button.Title" xml:space="preserve">
         <source>Start browsing</source>
         <target>Komenci retumi</target>
         <note>Text for a label that indicates the title of button from onboarding screen</note>
       </trans-unit>
+      <trans-unit id="Onboarding.DefaultBrowser.BottomButtonTitle.V2" xml:space="preserve">
+        <source>Skip</source>
+        <note>Text for a label that indicates the title of the bottom button from the default browser onboarding screen version 2.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.DefaultBrowser.FirstSubtitle.V2" xml:space="preserve">
+        <source>We clear your history when you close the app for extra privacy.</source>
+        <note>Text for a label that indicates the first subtitle for the default browser onboarding screen version 2.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.DefaultBrowser.SecondSubtitle.V2" xml:space="preserve">
+        <source>Make %@ your default to protect your data with every link you open.</source>
+        <note>Text for a label that indicates the second subtitle for the default browser onboarding screen version 2.</note>
```

Comment:
Thanks for the fixes. The placeholder here still needs to be explained in comments.
So adding: %@ is the name of the app (Focus/Klar)